### ISSUE TITLE
full set of superscript digits

### ISSUE THREAD

**Issue by Cogli** [psb1558/Elstob-font#14]
Given that Elstob is meant to be an academic typeface, it would be useful to include a full set of superscript digits, so commonly used to flag footnote.

**Comment by psb1558** [comment]
**Sigh.** I was hoping to get away with not doing this, but you're right: it needs to be done. Will work on it.

**Comment by psb1558** [comment]
I have roughed in superscript numbers for roman and italic and added a sups feature to provide access to them (onum, pnum, lnum, tnum, and zero have also been changed). I'll clean up the outlines and spacing as time permits, but in the meantime I'd appreciate it if anyone would try out this feature and report: do they look okay? are the weights right? does the OpenType work as expected? Both static (ttf, otf) and variable fonts have been rebuilt. Thanks

**Comment by Cogli** [comment]
I've tested the otfs using librewriter and lualatex.
As far as I see, sups feature seems to work fine. There is just an inconsistency with case feature.
I don't know how sups feature is supposed to work alongside case feature: presently, case feature changes oldstyle superscripts to lining _only in italic faces_ (and in any case doesn't seems to affect slashedzero - not even as a text digit, I mean). By the way, just a set of lining superscript would have been more than sufficient!

About the design: I'd just say that superscript numbers are just a little too dark, especially in heavier weights and especially 0s:
some less weight would probably improve readabilty.
[proof.pdf](https://github.com/psb1558/Elstob-font/files/5036654/proof.pdf)

**Comment by psb1558** [comment]
I learned in working with Junicode that some users wanted all the numeric styles in superscript. (I wouldn't have emitted a boldface sigh for just one style!). Fortunately I have RMX Tuner, which provides a head start--and of course there are lots of repeated outlines.

I've fixed case. Yesterday the weights looked fine, but today they look too heavy: what do you suppose happened to them overnight? I'll play with them till they're right. Probably post a fixed version tomorrow.

**Comment by Cogli** [comment]
> I've fixed case. Yesterday the weights looked fine, but today they look too heavy: what do you suppose happened to them overnight? I'll play with them till they're right. Probably post a fixed version tomorrow.

Too much staring, of course. I bet they look good on a regular page, as they normally seat in a meaningful text where they belong!

> I learned in working with Junicode that some users wanted all the numeric styles in superscript. (I wouldn't have emitted a boldface sigh for just one style!).

Tabular/proportional figures distinction comes for more or less compelling aesthetic and functional reasons. The same can be said about lining/oldstyle distinction — one can even use it to convey some semantical difference.
When it comes to superscripts, I'm not sure this all makes sense. Sure, someone may even find a use for smallcap superior numbers!

**Comment by psb1558** [comment]
I was thinking that it would be simplest to make the superscripts behave the way the rest of the figures behaved (with proportional-tabular and oldstyle-lining axes) and worried that doing it differently might needlessly complicate the OpenType code. In practice, though, it seems to work well enough to include just two styles of superscript and simply ignore the proportional-tabular axis in the feature code. So the default for superscripts is tabular lining, but onum will yield proportional oldstyle.

For color I've been testing with a random bit of prose with footnote-like numbers scattered throughout (via LibreOffice--it's appalling that one still can't do this in Word). I think it's starting to look okay, but I'm thinking of making the superscript oldstyle figures a tiny bit bigger:
[supstest.pdf](https://github.com/psb1558/Elstob-font/files/5040892/supstest.pdf)

**Comment by Cogli** [comment]
Looking at your supstest, zooming in and out, color looks definitely right.

There is something bothering me about oldstyle superscripts: superscripts are supposed to align with capital height, and that is why os-superscripts are in principle at least a questionable stylistic choice.
Looking at how that '9' pulls itself all that bit down, I wonder what they would look like if all os-superscripts were to be aligned on top.

**Comment by Cogli** [comment]
In my opinion, os-7 superscript probably needs some more weight: in 7.osf you have opted for a monolinear stem which when it comes to superscript becomes nearly impalpable.

**Comment by psb1558** [comment]
I've been wondering what to do about that seven. Might use a descending stem more like the lining seven.

It occurs to me that I've got a few nicely printed books with oldstyle footnote numbers. I'll study how they did it.

**Comment by psb1558** [comment]
Thinking aloud, here I've gathered footnote numbers from around one page of a well printed book from the Oxford Press (1953), and (being careful about scale and position) lined them up next to a word with an i and a cap for comparison.
<img width="1109" alt="image" src="https://user-images.githubusercontent.com/16215767/89650023-b95ca180-d88f-11ea-8344-3936573a764d.png">
I'd say, first, that these os superscripts are bigger than the ones in Elstob: I need to scale up by about 25%. Next, these figures have ascenders and descenders but relatively shorter than os figures in the main text, e.g.
<img width="448" alt="image" src="https://user-images.githubusercontent.com/16215767/89650712-d645a480-d890-11ea-8a28-3d44ddc3ef97.png">
Finally, the x-height of these os supersripts falls a little below the font's cap height, while high numbers like 6 project a little above. I'll check other books to see if the situation is similar--but now that I think about it, this type's handling of superscript figures is about what one would expect.

**Comment by Cogli** [comment]
Yes, reducing ascender and descender in superscripts sure is a good approach. Very much the same that happens, I would say, when one moves to a smaller optical size.

**Comment by psb1558** [comment]
Another try. Roman only at this point: italic will follow if I like the roman after a few days. Source is available, and static TrueType fonts in fonts/ttf/static.
[supstest.pdf](https://github.com/psb1558/Elstob-font/files/5045458/supstest.pdf)

**Comment by psb1558** [comment]
Italic, scaled to match the roman. Regular master:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/16215767/89831820-7408d000-db2c-11ea-971d-1ec8da503a01.png">
ExtraBold master:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/16215767/89831771-62272d00-db2c-11ea-935f-8d62988c0c85.png">

**Comment by alerque** [comment]
Some of these —most especially 5 but also to a lesser extent 6 and 9– look like they are raked quite a bit farther than the others. It's probably an optical illusion, but 5 seriously looks like it's falling off the back of the truck.

**Comment by psb1558** [comment]
I see it with the five, but not so much the six and nine. I wonder if those would look better if I shifted the weight a bit. Will experiment.

**Comment by alerque** [comment]
Also the 0 look a touch heavier or larger than the 1 in both italic weights shown.

**Comment by psb1558** [comment]
The 6 and 9 were already a little skewed in the roman original:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/16215767/89840022-2e083800-db3d-11ea-8cee-92b3f7b28a8b.png">
and that was accentuated in making the italic (for which I had no model). I've rotated the weight a little: will make further adjustments if necessary.

The 5 already had a distinct tilt in the roman face:
<img width="108" alt="image" src="https://user-images.githubusercontent.com/16215767/89840348-006fbe80-db3e-11ea-9380-6b9a6c63e668.png">
Shouldn't have skewed it so much in making the italic. I've corrected it by by about 15°.
<img width="454" alt="image" src="https://user-images.githubusercontent.com/16215767/89840561-87bd3200-db3e-11ea-9d71-cd08940cf9d8.png">
The oldstyle zero is surprisingly difficult. Will go on working on it.
